src/views/Settings.vue

```diff
@@ -46,7 +46,6 @@
             v-on:change="setShortenerProvider"
             v-model="shortenerProvider"
           >
-            <option>none</option>
```

Comment:
Should we not have the option to have no shortener configured?

Reply:
If that's the case, we could just use the quick links on home right?